setup.cfg

```diff
@@ -1,24 +1,25 @@
 [metadata]
 name = hoggorm
-version = attr: hoggorm.__version__
-description = Package for explorative multivariate statistics
-long_description = file: README.rst
-long_description_content_type = text/x-rst
-keywords =
+version=0.13.3
+description = Package for explorative multivariate statistics.
+author = Oliver Tomic
```

Comment:
I guess it is possible to insert multiple authors? Other important contributors are Kristian Hovde Liland, Thomas Graff and Andreas Fehlner.

Reply:
It seems that there is no standard way?
https://stackoverflow.com/questions/9999829/how-to-specify-multiple-authors-emails-in-setup-py

One could add an AUTHORS-File like 
https://github.com/cookiecutter/cookiecutter/blob/master/AUTHORS.md

Reply:
Authors.md looks like a nice way giving all authors credit. Let's implement this.